notebooks/work-in-progress/fuel-price-imputation/fuel-price-imputation-zane.ipynb

```diff
@@ -0,0 +1,1080 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.    frc.loc[:, string_cols] = frc[string_cols].fillna("NULL")
</pre><p>I think filling with "NULL" probably works since I've seen strings like "missing" be used. You could also add this step to a categorical column transformer pipeline by using <code>SimpleImputer</code> (with the <code>constant</code> strategy) as the "imputer" step.</p><p> </p><p>Not that familiar with pipelines but it seems like you might be able to move some of these  preprocessing steps into a pipeline for categorical cols and a pipeline for numerical cols.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1696/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I'm frustrated here because it seems like this isn't needed at all -- supposedly you can get the `OrdinalEncoder` to pass the NA values through, and the HistGBR model is supposedly fine with NA values on its own, but when I leave the NA values in there it complains about getting a mix of `NAType` and `str` values in the column.